### ISSUE TITLE
Running on Windows - `TypeError Cannot read property 'on' null`

### ISSUE THREAD

**Issue by 2shady4u** [josephbmanley/run-gut-tests-action#4]
Hello! I'm trying to get this action to work, but unfortunately it doesn't seem to work:
![image](https://user-images.githubusercontent.com/42484461/107570582-05472500-6bea-11eb-889a-082830553ed7.png)

Any idea what I'm doing wrong?

**Comment by yeslayla** [comment]
Hello @2shady4u :wave: 

So it looks like you are running on Windows and I haven't done much testing to make sure it works there.

This job depends on being able to run Linux docker images, so if your Windows agent either:
1. Doesn't have Docker CE installed [Runnable - Install Docker](https://runnable.com/docker/install-docker-on-windows-10)
2. Has Docker configured for Linux containers [Stack Overflow - Switch to Linux Containers](https://stackoverflow.com/questions/57081352/docker-command-line-to-switch-to-running-linux-containers-on-windows-core-machin)

I would suggest install Docker onto your agent or running your tests on the Linux agent.

**Comment by yeslayla** [comment]
@2shady4u If you have a specific project you are trying to implement this with, I can give it a look.

**Comment by 2shady4u** [comment]
Hello @josephbmanley 
Yes, I'm running this on a windows 10 host in Github Actions as found here:
https://github.com/2shady4u/fmod-gdnative/runs/1874701220?check_suite_focus=true

The YML for Github Actions can be found here:
https://github.com/2shady4u/fmod-gdnative/blob/e46b48244f0d7a7f921793f30bc784561b6f7d55/.github/workflows/all_builds.yml#L79

If I may ask, what platforms are supported?

**Comment by yeslayla** [comment]
So I've noticed.

Any platform that has docker installed is supported which includes Windows (Docker is not installed on base GitHub runner), Linux (Default GitHub runner has Docker installed), and Mac (I've not tested if the Mac GitHub runner has docker installed by default) Personally, I suggest just running on Linux since it is pre-configured and runs in a Linux container anyways.

It's also worth noting that the Docker image used has it's own version of Godot. So if you need to use your own binary, you would have to build your own container.

Simple `Dockerfile` Example:
```Dockerfile
FROM barichello/godot-ci:latest # Container containing Godot and required dependencies

COPY [ "MY GODOT INSTALL DIR/godot", "/usr/local/bin/godot" ] # Replaces installed version of Godot with yours

# If you have any other files that need to be alongside the godot binary, copy those as well
# This example (which is commented) copies the directory used by Godot Mono
# COPY [ "MY GODOT INSTALL DIR/GodotSharp", "/usr/local/bin/GodotSharp" ]
```

Then your `workflow.yaml`
```yaml
name: ci

on:
  push:
    branches: master

jobs:
  main:
    runs-on: ubuntu-latest
    steps:
      # Setup Docker build dependencies
      - name: Set up QEMU
        uses: docker/setup-qemu-action@v1
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v1

      # This builds your container
      - name: Build Docker Image
        id: docker_build
        uses: docker/build-push-action@v2
        with:
          push: true
          tags: GITHUB_PACKAGE_PATH:latest
          file: ./Dockerfile

      # Runs your tests with the new
      - uses: josephbmanley/run-gut-tests-action@v1.1.0
        with:
          directory: client
          containerImage: local/builder:latest
```

Though, I'll actually work on a new feature here to allow it to run locally.

**Comment by yeslayla** [comment]
Hey @2shady4u 

So `v1.1.0` and greater will now include my changes in #5 which should work on any platform with any Godot install, though it may require you to set the executable path.

Here's an example of that with a windows file path:
```yaml
      - name: Run GUT Tests
        uses: josephbmanley/run-gut-tests-action@v1.1.0
        with:
          directory: demo
          useContainer: false
          godotExecutable: "C:\\MyGodotInstallPath\\godot.exe"
```

Let me know if you run into any issues.

**Comment by yeslayla** [comment]
Closing this under presumption problem is solved by #5
